src/registry_schemas/example_data/schema_data.py

```diff
@@ -762,6 +762,8 @@
     'nameRequest': {
         'legalType': 'BC'
     },
+    'hasNameTranslations': True,
```

Comment:
Everything looks good to me so I will approve, but is there a reason to have this boolean? Wouldn't it be the same logic to have if json.get('nameTranslations') in any of our FE or BE code as if json['hasNameTranslations']?

Reply:
Removed that property